README.md

```diff
@@ -95,6 +95,18 @@ Read all visible cookies:
 Cookies.get(); // => { name: 'value' }
 ```
 
+*Note: It is not possible to read a particular cookie by passing one of the cookie attributes (which may or may not
+have been used when writing the cookie in question):*
+
+```javascript
+Cookies.get('foo', domain: 'sub.example.com'); // `domain` won't have any effect...!
+```
+
+That is, if a cookie with the name `foo` is visible from where `get()` was called,
+it's the one that is being read, no matter the domain or path. This also means that no guarantee
+can be made regarding which cookie is being retrieved in case one cookie is shadowing another,
+similarly named one with a differently scoped `domain` and/or `path` attribute.
```

Comment:
I'm not sure of the shadowing case should be documented here? Is it related to the issue we're trying to solve here? Have you tested it?

We have [an issue](https://github.com/js-cookie/js-cookie/issues/226) to document the shadowing cookie problem.